content/docs/rendering-elements.md

```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
-
-An element describes what you want to see on the screen:
+एलिमेंट्स एक रिएक्ट ऐप्स के सबसे छोटे बिल्डिंग ब्लॉक हैं।
 
+एक एलिमेंट वर्णन करता है कि आप स्क्रीन पर क्या देखना चाहते हैं:      
+
 ```js
 const element = <h1>Hello, world</h1>;
 ```
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+ब्राउज़र DOM एलेमेंट के विपरीत, React एलेमेंट्स एक सादा ऑब्जेक्ट है और इन्हें बनाना आसान है। React DOM React एलेमेंट्स के अनुसार DOM को अद्यतन बनाए रखता है।
 
->**Note:**
+>**टिप्पणी:**
 >
->One might confuse elements with a more widely known concept of "components". We will introduce components in the [next section](/docs/components-and-props.html). Elements are what components are "made of", and we encourage you to read this section before jumping ahead.
+> आप एलेमेंट्स को बहुचर्चित "कौम्पोनॅन्टस" समझ कर भ्रमित हो सकते है, हम "कौम्पोनॅन्टस" के बारे में [अगले भाग](/docs/components-and-props.html) में बतायेंगे। एलेमेंट्स वो है जिनका उपयोग करके एक कौम्पोनॅन्ट को बनाया जाता। आगे बढ़ने से पहले इस भाग को अवश्य पढ़े।
 
-## Rendering an Element into the DOM {#rendering-an-element-into-the-dom}
+## DOM में एक Element बनाना {#rendering-an-element-into-the-dom}
 
-Let's say there is a `<div>` somewhere in your HTML file:
+मान लीजिए की आपकी HTML फ़ाइल में कही एक `<div>` है:
 
 ```html
 <div id="root"></div>
 ```
 
-We call this a "root" DOM node because everything inside it will be managed by React DOM.
+हम इसे "रूट" DOM नोड कहते है क्यूँकि इसके अंदर की सारी चीज़ें React DOM के द्वारा संचालित होगी।
+
 
-Applications built with just React usually have a single root DOM node. If you are integrating React into an existing app, you may have as many isolated root DOM nodes as you like.
+React के उपयोग से बने ज़्यादातर ऐप्लिकेशन में एक ही रूट DOM नोड होता है। परंतु यदि आप React को किसी पहले से बने हुए app में उपयोग करना चाहते है तो आप कितने भी अलग अलग रूट DOM नोड बना सकते है।
 
-To render a React element into a root DOM node, pass both to `ReactDOM.render()`:
+एक React एलेमेंट को DOM नोड में दर्शाने के लिए, आपको इन दोनो को `ReactDOM.render()` में भेजना पड़ेगा:
 
 `embed:rendering-elements/render-an-element.js`
 
 [](codepen://rendering-elements/render-an-element)
 
-It displays "Hello, world" on the page.
-
-## Updating the Rendered Element {#updating-the-rendered-element}
+ये "नमस्ते दुनिया" को पेज पर दर्शाता है 
```

Comment:
I think we should not translate this `"नमस्ते दुनिया"` as in the code same its `Hello world`.

Reply:
Right

Reply:
`पर दर्शाता है ` => `पर दर्शाता है।`